docs/docs/tutorials/codealong/simple_dapp/0_project_setup.md

```diff
@@ -23,7 +23,7 @@ yarn init -yp
 3. Add the `aztec.js` and `accounts` libraries as a dependency:
 
 ```sh
-yarn add @aztec/aztec.js @aztec/accounts
+yarn add @aztec/aztec.js @aztec/accounts @aztec/noir-contracts.js
```

Comment:
Added this because it's used in the tests and its useful to show devs that they can use the `TokenContract` from this.